### PR TITLE
fix: fix deprecated sepolia subgraph url

### PR DESCRIPTION
**Pull request by wa0x6e**
Update ENS sepolia subgraph url

### CODE REVIEW

File: src/constants.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   },
   "ensSubgraph": {
     "1": "https://subgrapher.snapshot.org/subgraph/arbitrum/5XqPmWe6gjyrJtFn9cLy237i4cWw2j9HcUJEXsP5qGtH",
-    "11155111": "https://api.studio.thegraph.com/proxy/49574/enssepolia/version/latest"
+    "11155111": "https://subgrapher.snapshot.org/subgraph/arbitrum/DmMXLtMZnGbQXASJ7p1jfzLUbBYnYUD9zNBTxpkjHYXV"
```
Comment (bonustrack) [inline src/constants.json:17]: Looks like the right "official" endpoint is `https://api.studio.thegraph.com/query/49574/enssepolia/version/latest`

Comment (ChaituVR) [reply src/constants.json:17]: It is rate limited though

Comment (ChaituVR) [reply src/constants.json:17]: Yes yes, It's a tricky situation 

Official and Ratelimited
`https://api.studio.thegraph.com/query/49574/enssepolia/version/latest`

Not Official and not Ratelimited
`https://subgrapher.snapshot.org/subgraph/arbitrum/DmMXLtMZnGbQXASJ7p1jfzLUbBYnYUD9zNBTxpkjHYXV`

**Comment by ChaituVR** [comment]
> Can't we use our API key on `https://api.studio.thegraph.com/query/49574/enssepolia/version/latest`?

No, because it is not published, if we know the subgraph ID, it should work, I've asked about it on ENS discord but no one replied

**Comment by bonustrack** [comment]
Maybe ask TheGraph directly

File: src/constants.json (hunk 1 above)
Comment (wa0x6e) [reply src/constants.json:17]: Since we were using the official one before, does it mean it was also rate limited ?

Comment (ChaituVR) [reply src/constants.json:17]: Yes it was

Comment (ChaituVR) [reply src/constants.json:17]: Thegraph team confirmed that both are same subgraphs ✅ 

we can go with `https://subgrapher.snapshot.org/subgraph/arbitrum/DmMXLtMZnGbQXASJ7p1jfzLUbBYnYUD9zNBTxpkjHYXV`



> The subgraph ID of https://api.studio.thegraph.com/query/49574/enssepolia/version/latest is indeed DmMXLtMZnGbQXASJ7p1jfzLUbBYnYUD9zNBTxpkjHYXV

> You can see that the deployment ID it returns when you query the Studio endpoint above with the query below is the same as on 
https://thegraph.com/explorer/subgraphs/DmMXLtMZnGbQXASJ7p1jfzLUbBYnYUD9zNBTxpkjHYXV?view=About&chain=arbitrum-one

> Query:
```
{
 _meta{deployment} 
}
```

**Review by ChaituVR — APPROVED** [review]
utAck

**Pull request merged**